tests/src/python/test_qgsvectorlayertemporalproperties.py

```diff
@@ -754,6 +820,20 @@ def testExpressionMode(self):
         #
         # => start expression < end of range AND end expression > start of range
         self.assertEqual(props.createFilterString(context, range), '((to_datetime("my string field",  \'yyyy MM dd hh::mm:ss\')") < make_datetime(2020,5,6,8,9,10)) AND ((to_datetime("my end field",  \'yyyy MM dd hh::mm:ss\')") > make_datetime(2019,3,4,11,12,13))')
+        # map range              [-------------------------)
+        # feature ranges         .                         . [-------]  (false)
+        #                        .                         [-------]    (false)
+        #                        .                     [---.---]        (true)
+        #                        .            [-------]    .            (true)
+        #                        [-------]                 .            (true)
+        #                    [---.---]                     .            (true)
+        #                [-------]                         .            (true)
+        #          [-------]     .                         .            (false)
+        #
+        # => start expression <= end of range AND end expression > start of range
```

Comment:
From the diagram this should be:

```suggestion
        # => start expression < end of range AND end expression >= start of range
```